autorest/codegen/templates/operations_class.py.jinja2

```diff
@@ -0,0 +1,40 @@
+class {{ operation_group.class_name }}{{ object_base_class }}:
```

Comment:
same here, don't think we need to add a new template, just have the existing templates iterate over the list of operation groups

Reply:
It is extracted as part of `operations_container.py.jinja2`. I think it could make the structure more clean and easy to maintain.